sdks/go/pkg/beam/core/runtime/exec/translate.go

```diff
@@ -375,13 +375,15 @@ func (b *builder) makeLink(from string, id linkID) (Node, error) {
 
 			switch op {
 			case graph.ParDo:
-				n := &ParDo{UID: b.idgen.New(), PID: id.to, Inbound: in, Out: out}
+				n := &ParDo{UID: b.idgen.New(), PID: transform.UniqueName, Inbound: in, Out: out}
```

Comment:
Just checking, this change (id.to -> transform.UniqueName) is because the value of id.to is a name for the transform local to the link, right? i.e. it's not unique within the entire graph?

Reply:
Correct, and while this works OK, the unique names are unambiguous, and supposed to be required by the runner spec.